packages/components/breadcrumb/src/breadcrumb.tsx

```diff
@@ -69,13 +97,64 @@ export const Breadcrumb = forwardRef<BreadcrumbProps, "nav">((props, ref) => {
   const validChildren = getValidChildren(children)
   const count = validChildren.length
 
-  const cloneChildren = validChildren.map((child, index) =>
-    cloneElement(child, {
-      separator,
-      gap,
-      isLastChild: count === index + 1,
-    }),
-  )
+  const content = useMemo(() => {
+    const items = validChildren.map((child, index) =>
+      cloneElement(child, {
+        separator,
+        gap,
+        isLastChild: count === index + 1,
+      }),
+    )
+
+    if (!items) return null
+
+    if (count < maxItems) {
+      return items
+    }
```

Comment:
```suggestion
    if (count <= maxItems) {
      return items
    }
```
If `count` equals `maxItems`, not all breadcrumb items are shown. So, should the conditional branch include `=`, shouldn't it?